### PR TITLE
Finished Phase 2 of the Website Design

### PR DESCRIPTION
**Pull request by Medini-Rajendra**
## In this project, the following changes are done. 
- Added Readme File
- Checked for linter errors.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Medini Website Design</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+</head>
+<body>
+    <section class="container">
```
Comment (carlos-ssh) [inline index.html:12]: This ``section`` tag, should be a `` header``  tag. Kindly change it.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+    <section class="container mobilepara">
+        <h1>I'm Medini<br>Glad to see<br>you!</h1>
+        <p>I'm a software developer! I can help you
+            build a product, feature or website Look
+            through some of my work and 
+            experience! If you like what you see and
+            have a project you need coded, don't
+            hesitate to contact me.</p>
+        <h4>LET'S CONNECT</h4>
+        <!--img src='./images/social_media_icons.svg' class="social_media_icons" alt="social media icons"!>!-->
```
Comment (carlos-ssh) [inline index.html:27]: - Remove commented code.

File: index.html [hunk 3]
```diff
@@ -0,0 +1,170 @@
...
+    </section>
+    <section class="container mobilepara">
+        <h1>I'm Medini<br>Glad to see<br>you!</h1>
+        <p>I'm a software developer! I can help you
+            build a product, feature or website Look
+            through some of my work and 
+            experience! If you like what you see and
+            have a project you need coded, don't
+            hesitate to contact me.</p>
+        <h4>LET'S CONNECT</h4>
```
Comment (carlos-ssh) [inline index.html:26]: Change the font family:
- This is your requested style:


Screen Shot 2021-09-25 at 17 38 22

#
Yours:
Screen Shot 2021-09-25 at 17 39 25

#

File: index.html [hunk 4]
```diff
@@ -0,0 +1,170 @@
...
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
```
Comment (carlos-ssh) [inline index.html:91]: - Fix the requested design on your Figma template:
Screen Shot 2021-09-25 at 17 38 10
#
- Yours:
Screen Shot 2021-09-25 at 17 39 30

File: index.html (hunk 3 above)
Comment (carlos-ssh) [reply index.html:26]: - Should be a `` button`` or ``a`` tag.

File: index.html [hunk 5]
```diff
@@ -0,0 +1,170 @@
...
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
```
Comment (carlos-ssh) [inline index.html:55]: - Should be a `` button`` or ``a`` tag.

File: index.html [hunk 6]
```diff
@@ -0,0 +1,170 @@
...
```
Comment (carlos-ssh) [inline index.html:75]: - Should be a `` button`` or ``a`` tag.

File: index.html [hunk 7]
```diff
@@ -0,0 +1,170 @@
...
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
```
Comment (carlos-ssh) [inline index.html:73]: - Check the requested design. Is missing the font family.

File: index.html [hunk 8]
```diff
@@ -0,0 +1,170 @@
...
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
+                <li><h4>css</h4></li>
```
Comment (carlos-ssh) [inline index.html:92]: - Instead of ``h4`` should be an ``a`` tag.

File: index.html [hunk 9]
```diff
@@ -0,0 +1,170 @@
...
+                <li><h4>javaScript</h4></li>
+            </ul>
+            <h3>See Project</h3>
```
Comment (carlos-ssh) [inline index.html:95]: - Should be a `` button`` or ``a`` tag.

File: index.html [hunk 10]
```diff
@@ -0,0 +1,170 @@
...
```
Comment (carlos-ssh) [inline index.html:115]: - Should be a `` button`` or ``a`` tag.

File: index.html [hunk 11]
```diff
@@ -0,0 +1,170 @@
...
+            hesitate to contact me.</p>
+        <h4>LET'S CONNECT</h4>
+        <ul class="social_media_icons">
+            <li><a href=#><img src='./images/twitter.svg' alt="twitter icon"></a></li>
+            <li><a href=#><img src='./images/Linkedin_icon.svg' alt="LinkedIn icon"></a></li>
+            <li><a href=#><img src='./images/m_icon.svg' alt="M icon"></a></li>
+            <li><a href=#><img src='./images/cat_icon.svg' alt="Cat icon"></a></li>
+            <li><a href=#><img src='./images/hand_icon.svg' alt="Hand icon"></a></li>
+        </ul>
+        <h3>Get my Resume</h3>
```
Comment (carlos-ssh) [inline index.html:134]: - Should be a `` button`` or ``a`` tag.

File: index.html [hunk 12]
```diff
@@ -0,0 +1,170 @@
...
+                <ul>
+                    <li><img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;Back End Dev</li>
+                    <li>&nbsp;&nbsp;<img src='./images/Counter.svg' alt="Counter image"></li>
+                    <li>&nbsp;&nbsp;2015</li>
+                </ul>
+            </div>
+            <p>A daily selection of privately<br>personalized reads; no accounts or<br>sign-ups required.</p>
+            <ul class="diff_classes">
+                <li><h4>html</h4></li>
```
Comment (carlos-ssh) [inline index.html:71]: - Instead of ``h4`` should be an ``a`` tag.

File: index.html [hunk 13]
```diff
@@ -0,0 +1,170 @@
...
+                <li><h4>css</h4></li>
+                <li><h4>javaScript</h4></li>
```
Comment (carlos-ssh) [inline index.html:53]: - Instead of ``h4`` should be an ``a`` tag.

File: style.css [hunk 14]
```diff
@@ -0,0 +1,358 @@
...
+  font-weight: normal;
+  font-size: 16px;
+  line-height: 24px;
+}
+
+.social_media_icons {
+  list-style-type: none;
+
+  /* width: 50%;
+  height: 2em; */
```
Comment (carlos-ssh) [inline style.css:59]: [OPTIONAL]
- Remove commented code. Keep your project clean and professional.

File: style.css [hunk 15]
```diff
@@ -0,0 +1,358 @@
...
+  display: inline;
+  color: #6070ff;
+}
+
+.mobilepara > ul > li {
+  display: inline;
+  height: 2em;
+  width: 50%;
+  padding-left: 5%;
+}
```
Comment (carlos-ssh) [inline style.css:75]: [OPTIONAL]
**Don't Repeat Yourself**
- Keep your code optimized and reduce same lines of code.

File: style.css [hunk 16]
```diff
@@ -0,0 +1,358 @@
...
+  padding-top: 3%;
+  padding-bottom: 3%;
+}
+
+.about_section > ul > li {
+  display: inline;
+  height: 2em;
+  width: 50%;
+  padding-left: 5%;
+}
```
Comment (carlos-ssh) [inline style.css:88]: [OPTIONAL]
**Don't Repeat Yourself**
- Keep your code optimized and reduce same lines of code.

File: style.css [hunk 17]
```diff
@@ -0,0 +1,358 @@
...
+.item3 {
+  grid-column: 1 / span 2;
+  grid-row: 3;
+}
+
+.item4 {
+  grid-column: 1 / span 2;
+  grid-row: 4;
+}
+
```
Comment (carlos-ssh) [inline style.css:295]: [OPTIONAL]
**Don't Repeat Yourself**
- Keep your code optimized and reduce same lines of code.

File: style.css [hunk 18]
```diff
@@ -0,0 +1,358 @@
...
+.framework > i {
+  padding-top: 7%;
+  font-size: 23px;
+  padding-left: 40%;
+}
+
+.skill_div > i {
+  padding-top: 7%;
+  font-size: 23px;
```
Comment (carlos-ssh) [inline style.css:356]: [OPTIONAL]
**Don't Repeat Yourself**
- Keep your code optimized and reduce same lines of code.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @Medini-Rajendra , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html (hunk 1 above)
Comment (Medini-Rajendra) [reply index.html:12]: Changed section tag to header tag

File: index.html (hunk 2 above)
Comment (Medini-Rajendra) [reply index.html:27]: Removed commented code

File: index.html (hunk 3 above)
Comment (Medini-Rajendra) [reply index.html:26]: Changed font family. Also changed this to "a" tag.

File: index.html (hunk 4 above)
Comment (Medini-Rajendra) [reply index.html:91]: Fixed this issue.

File: index.html (hunk 5 above)
Comment (Medini-Rajendra) [reply index.html:55]: Modified it to "a" tag

File: index.html (hunk 6 above)
Comment (Medini-Rajendra) [reply index.html:75]: Modified it to "a" tag

File: index.html (hunk 7 above)
Comment (Medini-Rajendra) [reply index.html:73]: Changed it to the respective font as per the design

File: index.html (hunk 8 above)
Comment (Medini-Rajendra) [reply index.html:92]: Corrected it to "a" tag

File: index.html (hunk 9 above)
Comment (Medini-Rajendra) [reply index.html:95]: I have used the hover effect on the buttons as per the suggestion.

File: index.html (hunk 10 above)
Comment (Medini-Rajendra) [reply index.html:115]: I have used the hover effect on the buttons as per the suggestion.

File: index.html (hunk 9 above)
Comment (carlos-ssh) [reply index.html:95]: It's nice to know that you have been working on the requested changes :D... Remember implement the same colors for your fonts and hover effects in your backgrounds.

File: index.html (hunk 11 above)
Comment (Medini-Rajendra) [reply index.html:134]: Changed it to "a" tag

File: index.html (hunk 12 above)
Comment (Medini-Rajendra) [reply index.html:71]: Changed it to "a" tag

File: index.html (hunk 13 above)
Comment (Medini-Rajendra) [reply index.html:53]: Changed it to "a" tag

File: style.css (hunk 14 above)
Comment (Medini-Rajendra) [reply style.css:59]: Removed commented code

File: style.css (hunk 15 above)
Comment (Medini-Rajendra) [reply style.css:75]: This is optional comment. Will follow this in next project

File: style.css (hunk 16 above)
Comment (Medini-Rajendra) [reply style.css:88]: This is optional comment. Will follow this in next project

File: style.css (hunk 17 above)
Comment (Medini-Rajendra) [reply style.css:295]: This is optional comment. Will follow this in next project

File: style.css (hunk 18 above)
Comment (Medini-Rajendra) [reply style.css:356]: This is optional comment. Will follow this in next project

Comment (carlos-ssh) [reply style.css:387]: Nice to know that. I hope that this [article](https://medium.com/circuits-technical-talks/dry-css-9cbe3215af3d) it's gonna be helpful for you. Read it once that you have a chance.

**Review by carlos-ssh — APPROVED** [review]
Hi @Medini-Rajendra , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**